### ISSUE TITLE
$ in JSON path throws error

### ISSUE THREAD

**Issue by rmtuckerphx** [perpk/json-xform#31]
The `source` is valid JSON but the transform fails:

```
const jsonXform = require("@perpk/json-xform")
const source = {
    "name": 'Mark',
    "$job": 'programmer'
};

const mapping = {
    "fieldset": [
        {
            "from": "name"
        },
        {
            "from": "$job",
            "to": "myJob"
        }

    ]
};

// valid JSON
//source.$job
//source['$job']

jsonXform.mapToNewObject(source, mapping)
```

Error:

```
Parse error on line 1:\n$.$job\n--^\nExpecting 'STAR', 'I…IER', 'SCRIPT_EXPRESSION', 'INTEGER', 'END', got 'DOLLAR'
```

**Comment by perpk** [comment]
Thanks @rmtuckerphx for taking the time, will look at it asap!

**Comment by perpk** [comment]
@rmtuckerphx released in v1.0.1

Thanks again
Cheers!

**Comment by rmtuckerphx** [comment]
Thank you so much for the quick fix.
